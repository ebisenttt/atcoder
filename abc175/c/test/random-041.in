-20992436167816 343208168189505 847531006887358
