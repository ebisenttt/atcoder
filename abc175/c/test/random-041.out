826538570719542
